content/python.rst

```diff
@@ -106,6 +106,12 @@ NO
     other_bad = "string"
     some_bad = "string 'b' yes"
 
+* Use `from __future__ import unicode_literals` for Python3 compatibility.
```

Comment:
use where? how?

I think that we begin by using it in all tests... so also update the documentation so that all tests are done with unicode literals and absolute imports

the python-futures.org doc talks about cons and pros... but I think that we should also have at least a phrase arguing why we went for this